specs/Deployment/DeploymentAPI.md

```diff
@@ -184,7 +184,11 @@ they shut down, to refer to the updated framework package.
 ```C# (but really MIDL3)
 namespace Microsoft.Windows.ApplicationModel.WindowsAppRuntime
 {
+    [contractversion(2)]
+    apicontract Deployment{};
```

Comment:
`DeploymentContract`

Contract names end with `Contract`

Reply:
@DrusTheAxe, would DynamicDependency contract name also need to be updated likewise - https://github.com/microsoft/WindowsAppSDK/blob/main/dev/DynamicDependency/API/M.AM.DynamicDependency.idl ?

Reply:
Yes. At a minimum it'll need to change from `DynamicDependency` to `DynamicDependencyContract` comply with [specs/WinRT/WinRTAPIContracts.md](https://github.com/microsoft/WindowsAppSDK/pull/2293) section `2.5 Contract naming`

I wouldn't worry about code right now. For anything checked in let it sit. For anything pending let it wait (if it's just adding contracts). Once we sort out #2293 and get API Review signoff we can update code.